packages/volto/news/6481.bugfix

```diff
@@ -0,0 +1 @@
+Fixed Order blocks in sidebar doesn't respect layout restrictions by setting appropriate handlers in `Item.jsx`. @alihamza1221
```

Comment:
A good change log entry explicitly states what problem you fixed, not how you fixed it. The original message essentially said, "Fixed _____ by _writing code in a specific file_". See https://6.docs.plone.org/contributing/index.html#change-log-entry after the `Important` admonition for details.

```suggestion
While editing a block, and in the sidebar under the {guilabel}`Order` tab, when an item has {guilabel}`Required` checked, then it can no longer be deleted. Additionally, when  {guilabel}`Fixed position` is checked, the item's ordered position can no longer be changed. Previously neither option was honored. @alihamza1221
```